### DISCUSSION TITLE
Avoid to start from zero visits

### DISCUSSION THREAD

**Discussion by mimporto** [ibericode/koko-analytics#262] [Ideas]
Hi all. I'm currently using the plugin "SRS Simple hits Counter" to show the number of visits on the homepage of my website and I want to change it with Koko analytics. However I'd like to avoid to lose the number of visits that I currently have so I'm wondering if there is any way to start from a fixed number (the current number of visits on my site) and to add the new visits reported by Kiko analytics to this number.
Thanks, Domenico

**Comment by mimporto** [comment]
I solved updating the table wp_koko_analytics_site_stats with the sarting number of the visits on my site :-)

**Reply by dannyvankooten** [reply]
Hello @mimporto,

That's indeed the way to go - nicely done for figuring that out!
